src/main/containers/refresh-jit.js

```diff
@@ -0,0 +1,81 @@
+const AwsCredentials = require('../api/aws-credentials');
+const ResponseObj = require('../api/response');
+const Reloader = require('../api/reloader/reloader');
+
+const credentials = new AwsCredentials();
+
+async function refreshJitCallback(profileName, session) {
+  const refreshResponseObj = {
+    ...ResponseObj,
+    accountId: session.accountId,
+    roleName: session.roleName,
+    showRole: session.showRole,
+    profileName,
+  };
+
+  let creds = {};
+  await fetch(session.apiUri, {
+    method: 'GET',
+    headers: session.header,
+  }).then(async (response) => {
+    if (response.ok) {
+      creds = await response.json();
+    } else {
+      Manager.removeByName(profileName);
+      throw new Error('Unable to retrieve credentials from Divvy');
+    }
+  });
```

Comment:
You're mixing `async/await` with `Promise` syntax which isn't inherently wrong (`async/await` is just sugar over promises anyway) but is just kind of an odd way of doing things. I'd rewrite it like this:
```suggestion
  const response = await fetch(session.apiUri, {
    method: 'GET',
    headers: session.header,
  })
  
  if (response.ok) {
    creds = await response.json();
  } else {
    Manager.removeByName(profileName);
    throw new Error('Unable to retrieve credentials from Divvy');
  }
```

Might also want a `try/catch` block around each call that you have to `await` just in case the promise rejects so we can return an error properly.